src/index.html

```diff
@@ -10,23 +10,36 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <main class="main">
+      <form class="search-bar">
+        <label for="id1"></label>
+        <input
+          id="id1"
+          type="text"
+          data-qa="big"
+          action="#"
+          placeholder="Try “Los Angeles“"
+          class="search-bar__big input"
+        />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+        <label for="id2"></label>
```

Comment:
HTML Content: Labels should always have some text. If you want to hide it visually, but make it available for screen readers, you can use a technique like this: `<label class="visually-hidden" for="id2">Search</label>`. Class `visually-hidden` should be styled appropriately in CSS.